package.json

```diff
@@ -9,7 +9,7 @@
     "@semantic-release/github": "^10.0.3",
     "@semantic-release/npm": "^12.0.0",
     "@semantic-release/release-notes-generator": "^13.0.0",
-    "conventional-changelog-conventionalcommits": "^8.0.0",
+    "conventional-changelog-conventionalcommits": "^7.0.2",
```

Comment:
We should think of adding a test that ensures that semantic-release works with this config. I may give it a try / file it as an issue in the repo

Reply:
https://github.com/open-turo/semantic-release-config/issues/186